src/test-fbp/converter-error-expected.txt

```diff
@@ -0,0 +1 @@
+WRN: ./src/lib/common/sol-types.c:242 sol_irange_division() Division by zero: 10, 0
```

Comment:
That's a good improvement for expected file.

But in this case, I don't think we intend to spill that message for the regular user -- unless he asks for it, even less ensure that this is the exact message. The tests here that check the output should mostly check the output we give for regular (non-debugging) uses.

@bdilly what is going to be the default log level for "release builds"? I suggest we change the `src/test-fbp/run` to set this log level.